lnwire/accept_channel.go

```diff
@@ -292,3 +293,79 @@ func (a *AcceptChannel) MsgType() MessageType {
 func (a *AcceptChannel) SerializedSize() (uint32, error) {
 	return MessageSerializedSize(a)
 }
+
+// A compile time check to ensure AcceptChannel implements the TestMessage
+// interface.
+var _ TestMessage = (*AcceptChannel)(nil)
```

Comment:
This should be put in the test file like `message_test.go` or `test_message.go`?

Reply:
So move all the methods added to fulfill the new interface to `test_message.go`? Sure

Reply:
Done!